multidim-interop/impl/js/v0.46/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@libp2p/multidim-interop": "^1.0.0-c4eff4c5"
+  }
```

Comment:
```suggestion
  },
  "overrides": {
    "@libp2p/mplex": "9.0.5",
    "@libp2p/tcp": "8.0.5",
    "@libp2p/webrtc": "3.1.9",
    "@libp2p/websockets": "7.0.5",
    "@libp2p/webtransport": "3.0.9",
    "libp2p": "0.46.5"
  }
```

You should add this to pin the dependencies in the monorepo to the commit of the actual release.
Otherwise, you'll install a dirty `-c4eff4c5` for each package which is [evident in the lock file](https://github.com/libp2p/test-plans/blob/dc63bc5be8442dcf833a5c52c273bdde88d344f3/multidim-interop/impl/js/v0.46/package-lock.json).
```
        "@libp2p/mplex": "9.0.5-c4eff4c5",
        "@libp2p/tcp": "8.0.5-c4eff4c5",
        "@libp2p/webrtc": "3.1.9-c4eff4c5",
        "@libp2p/websockets": "7.0.5-c4eff4c5",
        "@libp2p/webtransport": "3.0.9-c4eff4c5",
        "@multiformats/mafmt": "^12.1.2",
        "@multiformats/multiaddr": "^12.1.5",
        "aegir": "^40.0.12",
        "libp2p": "0.46.9-c4eff4c5",
```
this may not seem like a problem but I don't think it's good practice to test against unreleased commits.

adding these overrides at least allows us to explicitly specify which version of libp2p packages are being tested without having to search in the lock file